### DISCUSSION TITLE
A. AB Balance

### DISCUSSION THREAD

**Discussion by BerentKaya** [sonsuzus/sonsuzus#532] [Soru Cevap]
Bir dizi s veriliyor, s'nin uzunluğu n ve yalnızca a ve/veya b karakterlerinden oluşuyor.

AB(s) ifadesi, s dizisinde ab alt dizesinin meydana gelme sayısını temsil eder. Aynı şekilde, BA(s) ifadesi de s dizisinde ba alt dizesinin meydana gelme sayısını temsil eder.

Bir adımda, herhangi bir i indeksini seçebilir ve si değerini a veya b karakteriyle değiştirebilirsiniz.

AB(s)=BA(s) eşitliğini sağlamak için kaç adımda minimum durumda olacağınızı öğrenmek istiyorsunuz.

Hatırlatma:

d alt dizisinin s dizisi içinde meydana gelme sayısı, d ile eşit olan s[i]...s[i+|d|-1] alt dizesinin bulunduğu i indeksleri (1≤i≤|s|−|d|+1) sayısıdır. Örneğin, AB(aabbbabaa)=2'dir çünkü aabbbabaa dizisinde i=2 ve i=6'da iki indeks vardır.

Giriş
Her test birden fazla test durumunu içerir. İlk satır test durumlarının sayısını t (1≤t≤1000) belirtir. Test durumlarının açıklamaları aşağıda verilmiştir.

Her test durumunun tek satırı, a ve/veya b karakterlerinden oluşan tek bir dizi s içerir (1≤|s|≤100, burada |s| s dizisinin uzunluğudur).

**Comment by Sevvalgocgun1** [comment]
t = int(input())
for _ in range(t):
    s = input().strip()
    if s.count("ab") == s.count("ba"):
        print(0)
    else:
        print(1)
